pkg/telemetryquerylanguage/tql/README.md

```diff
@@ -0,0 +1,167 @@
+# Telemetry Query Language
+
+The Telemetry Query Language is a query language for transforming open telemetry data based on the [OpenTelemetry Collector Processing Exploration](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/processing.md).
+
+This package reads in TQL queries and converts them to invokable conditions and functions based on the TQL's grammar.
+
+## Grammar
+
+The TQL grammar includes Invocations, Values and Conditions.  The TQL makes an assumption that queries are separated by signal; there is no way in the grammar to specify a specific signal.
```

Comment:
> The TQL makes an assumption that queries are separated by signal

It's not clear to me what this means. For example, later in the docs we have an example of a query for each signal type, which _appears_ to be the same query:
```
traces:
  delete(attributes["http.request.header.authorization"])
metrics:
  delete(attributes["http.request.header.authorization"])
logs:
  delete(attributes["http.request.header.authorization"])
```

Is there a way to clarify this statement?

Reply:
I removed that statement and added a clarifying paragraph at the start of the readme.  I think some of the holes will get filled in when the 3 signal's contexts are moved into the module.  Then I'll be able to link between docs.